apps/example-orchestrator-ui/components/Subscriptions/Subscriptions.tsx

```diff
@@ -56,82 +62,105 @@ export const Subscriptions: FC<SubscriptionsProps> = ({
 }) => {
     const router = useRouter();
     const { theme } = useOrchestratorTheme();
+    const hiddenColumns: Array<keyof Subscription> = [
+        'organisationName',
+        'productName',
+    ];
 
-    const tableColumnConfig: TableColumns<Subscription> = {
+    const tableColumns: TableColumns<Subscription> = {
+        subscriptionId: {
+            field: 'subscriptionId',
+            name: 'ID',
+            width: '100',
+            render: (value: string) => sliceUuid(value),
+        },
         description: {
-            displayAsText: 'Description',
-            initialWidth: 400,
-            renderCell: (cellValue, row) => (
-                <Link href={`/subscriptions/${row.subscriptionId}`}>
-                    {cellValue}
+            field: 'description',
+            name: 'Description',
+            width: '400',
+            render: (value: string, record) => (
+                <Link href={`/subscriptions/${record.subscriptionId}`}>
+                    {value}
                 </Link>
             ),
         },
+        status: {
+            field: 'status',
+            name: 'Status',
+            width: '110',
+            render: (value: string) => (
+                <SubscriptionStatusBadge subscriptionStatus={value} />
+            ),
+        },
         insync: {
-            displayAsText: 'In Sync',
-            initialWidth: 110,
-            renderCell: (cellValue) =>
-                cellValue ? (
+            field: 'insync',
+            name: 'In Sync',
+            width: '110',
+            render: (value: boolean) =>
+                value ? (
                     <CheckmarkCircleFill color={theme.colors.primary} />
                 ) : (
                     <MinusCircleOutline color={theme.colors.mediumShade} />
                 ),
         },
         organisationName: {
-            displayAsText: 'Customer Name',
-            isHiddenByDefault: true,
+            field: 'organisationName',
+            name: 'Customer Name',
         },
         organisationAbbreviation: {
```

Comment:
organisationAbbreviation will not exists in the GPL version.

Feel free to ignore the code change; but than I expect a ticket.

Reply:
Also good too use the new convention when doing something with org:

I will probably add a "customer" in the optional subscription detail type. With a mandatory name when the object is provided. E.g. rename "organisation" to "customer" as discussed in our meeting.